FirebaseCore/Sources/FIRHeartbeatLogger.m

```diff
@@ -87,7 +87,8 @@ - (FIRHeartbeatsPayload *)flushHeartbeatsIntoPayload {
 }
 
 - (void)flushHeartbeatsIntoPayloadWithCompletionHandler:
-    (void (^)(FIRHeartbeatsPayload *))completionHandler {
+    (void (^)(FIRHeartbeatsPayload *))completionHandler
+    API_AVAILABLE(ios(13.0), macosx(10.15), macCatalyst(13.0), tvos(13.0), watchos(6.0)) {
```

Comment:
Above in this file is another method that is gated on the corresponding method's availability in the header: https://github.com/firebase/firebase-ios-sdk/blob/3e9c506844eff2d6bc0ea45fb6c0f2d295bdb220/FirebaseCore/Sources/FIRHeartbeatLogger.m#L77-L78

Reply:
edit. does not fix the issue because the API in the implementation won't be found. The `API_AVAILABLE` attribute does not change that the method is still compiled for iOS 12.

Reply:
This individual change is irrelevant to the fix, but is probably fine for consistency's sake.